src/w3c/templates/show-people.js

```diff
@@ -1,60 +1,63 @@
 export default (conf, name, items = []) => {
   const html = hyperHTML;
   // stuff to handle RDFa
-  const attr = Object.freeze(conf.doRDFa ? {
-    ...name === "Editor" ? {
-      rp1: "rdf:first",
-      rp2: "foaf:Person"
-    } : name === "Author" ? {
-      rp1: "dc:contributor",
-      rp2: "foaf:Person"
-    } : {},
-    rn: "foaf:name",
-    rm: "foaf:mbox",
-    rwu: "foaf:workplaceHomepage",
-    rpu: "foaf:homepage",
-    propSeeAlso: "rdfs:seeAlso"
-  } : {});
```

Comment:
I would keep `attr` here. It makes `attr.rp`, for example, to be `undefined` when `conf.deRDFa` is falsey, and that helps hyperHTML by explicitly asking attribute removal:

```js
html`<span property=${undefined} />`
```

This is exactly same as:

```js
html`<span />` // .hasAttribute("property") will return false for both examples
```

This enables us to keep if-statements minimal.

Reply:
Good to know, I'll go back to the old way.